src/main/java/Gicha/repository/CharacterRepository.java

```diff
@@ -0,0 +1,6 @@
+package Gicha.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface CharacterRepository extends JpaRepository<Character, Integer> {
```

Comment:
Javadoc pro repository a service je dost casto vhodny.

Reply:
jenom pro repo a service?

Reply:
na DTO to vetsinou neni moc potreba, controller je jasnej, ale v repu a service je business kod, ten co neco opravdu dela ..